File: src/main/java/com/lessmarkup/interfaces/system/MailTemplateModel.java
package com.lessmarkup.interfaces.system;

public class MailTemplateModel {
    private String subject;
    private String userEmail;
    private String userName;

    public String getSubject() {
        return subject;
    }
    public void setSubject(String subject) {
        this.subject = subject;
    }

    public String getUserEmail() {
        return userEmail;
    }
    public void setUserEmail(String userEmail) {
        this.userEmail = userEmail;
    }

    public String getUserName() {
        return userName;
    }
    public void setUserName(String userName) {
        this.userName = userName;
    }
}

File: src/main/scala/com/lessmarkup/interfaces/system/MailTemplateModel.scala
/*
 * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
 * If a copy of the MPL was not distributed with this file, You can obtain one at
 * http://mozilla.org/MPL/2.0/.
 */

package com.lessmarkup.interfaces.system

class MailTemplateModel {
  private var subject: String = null
  private var userEmail: String = null
  private var userName: String = null

  def getSubject: String = subject

  def setSubject(subject: String) {
    this.subject = subject
  }

  def getUserEmail: String = userEmail

  def setUserEmail(userEmail: String) {
    this.userEmail = userEmail
  }

  def getUserName: String = userName

  def setUserName(userName: String) {
    this.userName = userName
  }
}